pkg/api/defaults-kubelet.go

```diff
@@ -161,13 +161,13 @@ func (cs *ContainerService) setKubeletConfig() {
 			for key, val := range staticWindowsKubeletConfig {
 				profile.KubernetesConfig.KubeletConfig[key] = val
 			}
+		} else {
+			for key, val := range staticLinuxKubeletConfig {
+				profile.KubernetesConfig.KubeletConfig[key] = val
+			}
 		}
 
 		setMissingKubeletValues(profile.KubernetesConfig, o.KubernetesConfig.KubeletConfig)
-		// The previous call may have copied values from the orchestartor profile, which are not valid on Windows
```

Comment:
We don't need this deletion anymore as the `--pod-manifest-path` key is being set to `""` in the Windows static kublet config set, and those keys are now being deleted in the `removeKubeletFlags` convenience func